easybuild/easyconfigs/s/Stacks/Stacks-1.19-goolf-1.4.10.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+#
+# Full Stacks pipeline is php + mysql + command line utilies
+# This will install JUST the command line utilities
+
```

Comment:
the above has been added; please recheck zlib need and test/merge as appropriate